core/src/components/popover/popover.ios.scss

```diff
@@ -35,6 +35,7 @@
   height: 10px;
 
   overflow: hidden;
+  z-index: 12;
```

Comment:
Can we add a comment here describing why this is necessary? Also is there a reason why we specifically chose 12?

Reply:
Linking my [comment](https://github.com/ionic-team/ionic-framework/pull/28986#discussion_r1481793254) in this thread since it's related.